### PR TITLE
Starter kit

### PR DESCRIPTION
**Pull request by justin0022**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,11 +33,11 @@ These instructions will get you a copy of the project up and running on your loc
 
 - TODO: add link to quiz to request Canvas events data and add this data to the data folder
 - review [data_dictionary](./data_dictionary.md)
```
Comment (justin0022) [inline README.md]: These lines should be deleted before merge

File: README.md [hunk 2]
```diff
@@ -48,11 +48,18 @@ These instructions will get you a copy of the project up and running on your loc
 1. Open terminal on Mac or command line on Windows. We like to use [VSCode's integrated terminal](https://code.visualstudio.com/docs/editor/integrated-terminal), as it works for both Mac and PC.
 1. Clone repo. `git clone {paste URL you copied}`
 1. Navigate into repo. `cd hack-la-203`
-1. Create a folder using your group name. All of your work should be contained inside this folder to reduce the likelihood of merge conflicts when you submit your work later. 
-1. Copy the [final_submission_template.md](./final_submission_template.md) into the folder you created.
+1. Rename the `src_rename_me` folder with your group name. All of your work should be contained inside this folder to reduce the likelihood of merge conflicts when you submit your work later. 
+1. Download your events data from the Canvas quiz and place under the `data` folder, and rename the file as `events.csv`. Anything that is contained in this folder is gitignored.
```
Comment (alisonmyers) [inline README.md]: can the data folder be in the src_rename_me template with a .gitignore file within.

**Review by alisonmyers — CHANGES_REQUESTED** [review]
1) add data folder to src_rename_me with .gitignore
```
# Ignore everything in this directory
*
# Except this file
!.gitignore
```

2) include instructions for not committing data in Jupyter notebooks 

3) add .gitignore with
- .env
- data/* 
- etc.

File: README.md (hunk 2 above)
Comment (justin0022) [reply README.md]: I moved the `data` folder to inside `read_me` now.

**Comment by justin0022** [comment]
> 1. add data folder to src_rename_me with .gitignore
> 
> ```
> # Ignore everything in this directory
> *
> # Except this file
> !.gitignore
> ```
> 
> 2. include instructions for not committing data in Jupyter notebooks
> 3. add .gitignore with
> 
> * .env
> * data/*
> * etc.

I believe these are all addressed. I added a general gitignore from https://gist.github.com/octocat/9257657

**Review by alisonmyers — APPROVED** [review]

**Comment by alisonmyers** [comment]
LGTM.

**Pull request merged**